python/paddle/distributed/launch.py

```diff
@@ -212,12 +240,25 @@ def start_procs(args):
                 logger.warning("Use Cloud specified port:{}.".format(
                     cloud_paddle_port))
 
+    free_ports = None
+    if not args.use_paddlecloud and num_nodes <= 1:
```

Comment:
only use in single-node?

Reply:
Yes. This PR will only resolve the single node and when they use multi-nodes, the cluster-system will assign the property port.